extensions/sendbird/v1/actions/getUser/getUser.ts

```diff
@@ -0,0 +1,90 @@
+import { z, ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+import { type Action } from '@awell-health/extensions-core'
+import { fromUnixTime } from 'date-fns'
+import { type settings } from '../../../settings'
+import { Category, validate } from '@awell-health/extensions-core'
+import { SettingsValidationSchema } from '../../../settings'
+import { FieldsValidationSchema, fields, dataPoints } from './config'
+import {
+  SendbirdClient,
+  isSendbirdChatError,
+  sendbirdChatErrorToActivityEvent,
+} from '../../client'
+
+export const getUser: Action<typeof fields, typeof settings> = {
+  key: 'getUser',
+  title: 'Get user',
+  description: 'Gets user using Chat API.',
+  category: Category.COMMUNICATION,
+  fields,
+  dataPoints,
+  previewable: false,
+  onActivityCreated: async (payload, onComplete, onError) => {
+    try {
+      const {
+        settings: { applicationId, chatApiToken, deskApiToken },
+        fields: { userId },
+      } = validate({
+        schema: z.object({
+          settings: SettingsValidationSchema,
+          fields: FieldsValidationSchema,
+        }),
+        payload,
+      })
+
+      const client = new SendbirdClient({
+        applicationId,
+        chatApiToken,
+        deskApiToken,
+      })
+
+      const res = await client.chatApi.getUser(userId)
+
+      await onComplete({
+        data_points: {
+          nickname: res.data.nickname,
+          accessToken: res.data.access_token,
+          isActive: String(res.data.is_active),
+          createdAt: fromUnixTime(res.data.created_at).toISOString(),
+          lastSeenAt: fromUnixTime(res.data.last_seen_at).toISOString(),
+          hasEverLoggedIn: String(res.data.has_ever_logged_in),
+          metadata: JSON.stringify(res.data.metadata),
```

Comment:
Is there a reason that in `getCutomer.ts` you are pre-processing custom fields of a customer (which is similar to metadata of a user) and you are not doing that here?

```javascript
      const customFields = res.data.customFields?.reduce((acc, curr) => {
        return { ...acc, [curr.key]: curr.value }
      }, {})
```

Reply:
Structure difference, see [Slack comment](https://htdhealth.slack.com/archives/C04RZELCC1G/p1691743392835329?thread_ts=1691741162.871479&cid=C04RZELCC1G)